util/mock-style.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
Can we move this into common-config? `util` wasn't meant to contain source code, but rather be a parent folder for utility packages.